third-party/realdds/include/realdds/dds-stream-profile.h

```diff
@@ -85,6 +88,8 @@ class dds_stream_profile
         , _frequency( frequency )
     {
     }
+    dds_stream_profile( dds_stream_profile const & ) = default;

```

Comment:
Why do you need this? It won't work once we have the `weak_ptr` in there. Anything designed to work under `shared_ptr` shouldn't need this.

Reply:
I don't. I used `auto` to iterate over a vector of profiles and it needed a default constructor. Will fix to `auto &`